OSS Project Pricing Page

zchase opened the pull request:
This PR introduces an OSS Pricing page to pulumi.com. The intent is for someone to submit the form to get in contact with us about a free OSS license.

@susanev I think we will need to update the pricing page as well to merge this in. My assumption is that the pricing page is where we would link to this page from but interested to hear if you think there is a better place.

susanev commented on content/pricing/oss-project.md:
```diff
@@ -0,0 +1,47 @@
+hero:
+    title: Pulumi Open Source Program
+    description: |
+        At the core of Pulumi is an open source project and an engaged community. Pulumi exists thanks
```
this opening didn't really tell me what is available in a clear way. anyway to shorten this, and be specific that folks can get access to a paid plan for free?

zchase replied:
👍

susanev commented on content/pricing/oss-project.md:
```diff
@@ -0,0 +1,47 @@
+        deeply committed to being a good steward of the Pulumi open source project. As a way to
+        give back to the community, the Pulumi Open Source Program was created to help open source
+        projects and teams innovate and scale faster through cloud engineering.
+    cta_text: Request Access
```
```suggestion
    cta_text: Apply
```

i think `Apply` or `Submit Application` may work better here

zchase replied:
👍

susanev commented on content/pricing/oss-project.md:
```diff
@@ -0,0 +1,47 @@
+features:
+    title: Program Features
+    description: |
+        Projects can utilize the Pulumi open source version today for free if they prefer to manage
```
I think this sentence should move to the end or maybe even be removed? we should lead with what they are getting as part of this program

zchase replied:
👍

susanev commented on content/pricing/oss-project.md:
```diff
@@ -0,0 +1,47 @@
+contact:
+    title: Application Process
+    description: |
+        1. Submit the application form on the right side of this page to help us verify that your project meets all program requirements.
```
on mobile or smaller screens the form isn't on the right side, in general, we should try and avoid directional words like this

zchase replied:
👍

susanev commented on content/pricing/oss-project.md:
```diff
@@ -0,0 +1,47 @@
+    title: Application Process
+    description: |
+        1. Submit the application form on the right side of this page to help us verify that your project meets all program requirements.
+        2. You can expect an initial response from our team within 5-10 business days. You may be asked to provide additional information
```
this seems long just for an initial response? are we expecting a huge influx of applications?

aaronkao replied:
Why don't we set it as 3-5 business days. If we receive an influx, we can up the time.

zchase replied:
👍

susanev commented on content/pricing/oss-project.md:
```diff
@@ -0,0 +1,47 @@
+        If your application is accepted, we will work with you to enroll your account into the Program. Your application must be renewed annually to make sure that your project still meets the Program requirements, which may change from year to year. If you no longer qualify for renewal, we will work with you to make any needed transition as smooth as possible.
+
+    form:
+        headline: Request Access
```
```suggestion
        headline: Open Source Program Application
```

request access seems really strange as a heading here, this is the application right?

aaronkao replied:
Yeah, I think that was copied over from another web form template. Apply is the correct verb.

zchase replied:
👍

susanev reviewed (commented):
im still thinking through where to link to this form the pricing page, will follow up later today. but im including some content feedback now.

im also a little bit worried about the name of this program, could we have a more distinct name from the open source offering?

cnunciato reviewed (commented):
Some initial feedback for ya. I do think we should consider naming this something else, though, as it's hard to imagine it not getting confused with our current "open source" offering.  

Other questions we should probably answer on this page are whether the Pulumi organization can be created now (e.g., as a new trial), needs to be created later, whether existing nonprofits can opt into the program after having previously converted, and so on.

cnunciato commented on content/pricing/oss-project.md:
```diff
@@ -0,0 +1,47 @@
+hero:
+    title: Pulumi Open Source Program
+    description: |
+        At the core of Pulumi is an open source project and an engaged community. Pulumi exists thanks
```
☝️ Suggest removing the first few sentences here as they don't seem to be super relevant.

zchase replied:
👍 I reworked the paragraph. Let me know what you think

aaronkao commented on content/pricing/oss-project.md:
```diff
@@ -0,0 +1,47 @@
+    title: Program Requirements
+    description: |
+        **Not seeking profit** --- Your project must be part of an open source foundation that does not seek to make a profit
+        through services. The organization must be a valid 501c3 organization with a Employer
```
It's a good question. I think the criteria should be you are part of a nonprofit organization that has status and the code you are committing holds an acceptable open source license. For profit companies that have open source projects would not qualify. 

Any means is probably more comprehensive

zchase replied:
👍

zchase commented on content/pricing/oss-project.md:
```diff
@@ -0,0 +1,47 @@
+
+    form:
+        headline: Request Access
+        hubspot_form_id: 7465fe09-5022-4f43-b86f-3a571da9106a
```
👍

zchase commented on content/pricing/oss-project.md:
```diff
@@ -0,0 +1,47 @@
+        through services. The organization must be a valid 501c3 organization with a Employer
+        Identification Number (EIN).
+
+        **OSI-approved open source license** --- Your project must be published under an OSI-approved open source license.
```
👍

zchase commented on content/pricing/oss-project.md:
```diff
@@ -0,0 +1,47 @@
+        1. Submit the application form on the right side of this page to help us verify that your project meets all program requirements.
+        2. You can expect an initial response from our team within 5-10 business days. You may be asked to provide additional information
+
+        If your application is accepted, we will work with you to enroll your account into the Program. Your application must be renewed annually to make sure that your project still meets the Program requirements, which may change from year to year. If you no longer qualify for renewal, we will work with you to make any needed transition as smooth as possible.
```
@aaronkao Do you have any suggestions for this section?

cnunciato commented:
I'll hold off on reviewing this further for the 👍 from @aaronkao, as I think there are still some questions to be answered.

susanev commented:
@zchase @aaronkao i still have a few concerns about this page...
- the name of this project, is really similar to other names and im worried it wont be clear what this is in relation to the plans on the pricing page
- we don't have a sku currently that has a 25 member limit, and unlimited stacks, will we need to create one for this or will things get turned on manually for folks who are accepted into this program?
- for the URL can we name it the same as whatever the project ends up getting named?

susanev commented on content/pricing/oss-project.md:
```diff
@@ -0,0 +1,49 @@
+    title: Application Process
+    description: |
+        Submit the application form on this page. You can expect an initial response within
+        2-5 business days, at which time we may ask you to provide some additional information
```
```suggestion
        2 – 5 business days, at which time we may ask you to provide some additional information
```

zchase replied:
👍

susanev commented:
@zchase for the linking from the pricing page, lets add a FAQ, something like ...

```
How can organizations that are not seeking a profit, and projects under an OSI-approved open source license use Pulumi for free?
```

that links to the page with the details.

zchase commented:
@susanev I updated the URL to match better with the name of the page. 

- I think it will be fine to leave the title as-is for now, but leave the option to change it later if we hear confusion from customers.
- Onboarding these folks will be a manual process for now, but essentially this the current Team PRO SKU w/ 100% discount. I believe we have already onboarding 1 customer with this plan, so it might exist in some capacity already?
- Updated

Also if you don't mind, let's do the pricing page changes in a different PR. I think we will likely want to add an FAQ section and it would be to spend at least a little time thinking through exactly what we want to write and present on the /pricing page.

aaronkao commented:
> @zchase @aaronkao i still have a few concerns about this page...
> 
> * the name of this project, is really similar to other names and im worried it wont be clear what this is in relation to the plans on the pricing page
> * we don't have a sku currently that has a 25 member limit, and unlimited stacks, will we need to create one for this or will things get turned on manually for folks who are accepted into this program?
> * for the URL can we name it the same as whatever the project ends up getting named?



> @zchase @aaronkao i still have a few concerns about this page...
> 
> * the name of this project, is really similar to other names and im worried it wont be clear what this is in relation to the plans on the pricing page
> * we don't have a sku currently that has a 25 member limit, and unlimited stacks, will we need to create one for this or will things get turned on manually for folks who are accepted into this program?
> * for the URL can we name it the same as whatever the project ends up getting named?

- I agree the name may seem confusing, but there is no confusion that this is free stuff for OSS projects. We would probably just link to this at the very bottom of the /pricing page with a clear description. We could also call this "OSS Free Offer". Do you have any suggestions?
- This is just the the maxed version of the current Pro plan - https://www.pulumi.com/pricing/
- Yes, of course

susanev commented:
oh for the free plan my concern is just the confusion between the free plan currently: community and this free plan.
